src/app/file-manager/transferables-grid/transferables-grid.component.ts

```diff
@@ -1,27 +1,29 @@
-import { Component, OnInit, AfterViewInit, Input, ViewChild, ChangeDetectionStrategy } from '@angular/core';
+import { Component, OnInit, AfterViewInit, Input, ViewChild, ChangeDetectionStrategy, NgZone } from '@angular/core';
```

Comment:
Unused imports should be removed here as well. The IDE should be able to show which imports aren't being used, at least Intellij can do this.